content/docs/examples/kubeflow-samples.md

```diff
@@ -0,0 +1,53 @@
++++
+title = "Kubeflow samples"
+description = "How to access the Kubeflow web UIs"
```

Comment:
Please change the description to something like: 

Examples that demonstrate machine learning use cases with Kubeflow